src/app.py

```diff
@@ -705,16 +705,30 @@ def reindex_all(self):
 
         try:
             translator = self.init_translator(token)
-            threading.Thread(target=translator.translate_all, args=[]).start()
+            job_name = 'reindex_all'
+            if self._is_job_thread_running(job_name) is False:
+                threading.Thread(target=translator.translate_all, args=[], name=job_name).start()
+                logger.info('Started live reindex all')
+            else:
+                logger.info('Could not start live reindex all. Thread already running.')
+                return 'Request of live reindex all documents not accepted', 409
```

Comment:
@libpitt I would change the return to add that the method is already running.

"Request of live reindex all documents not accepted. Reindex all process already running" - something like that